app/src/main/java/com/duckduckgo/app/autocomplete/api/AutoComplete.kt

```diff
@@ -468,3 +466,15 @@ class AutoCompleteApi @Inject constructor(
         val score: Int = DEFAULT_SCORE,
     )
 }
+
+@VisibleForTesting
```

Comment:
Do we need the annotation since this is internal?

Reply:
The annotation is there to mark the fact that this function should be private, but the visibility was changed for testing purposes. I didn't add what the visibility should have been. By default, if not mentioned,  is private (as in this case).

`@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)`